fedora/client/openidbaseclient.py

```diff
@@ -170,6 +170,7 @@ def __init__(self, base_url, login_url=None, useragent=None, debug=False,
         # Also hold on to retry logic.
         # http://www.coglib.com/~icordasc/blog/2014/12/retries-in-requests.html
         server_errors = [500, 501, 502, 503, 504, 506, 507, 508, 509, 599]
+        method_whitelist = Retry.DEFAULT_METHOD_WHITELIST.union(set(['POST']))
```

Comment:
Ok NM, seeing the comment on https://github.com/fedora-infra/python-fedora/issues/144#issuecomment-160739487 ``Retry.DEFAULT_METHOD_WHITELIST`` is a frozenset and therefore does not have a ``.add()``. Tbh, I'm even surprised it supports ``.union()`` and this makes it modifiable.